_lzbench/lzbench.cpp

```diff
@@ -966,5 +997,6 @@ int main( int argc, char** argv)
     else
 #endif
         free((void*)inFileNames);
+        free(cpu_brand);
```

Comment:
Please judge that the pointer is not empty before free.
if(cpu_brand){
    free(cpu_brand);
}

Reply:
Done.
I still don't get that calloc() thing, can you explain why do you think calloc() is better than malloc()?